test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -361,6 +361,7 @@ public void run() {
 
         private boolean single = true;
         private boolean tlsEnabled = false;
+        private boolean autoCommit = false;
```

Comment:
Can we use noAutoCommit to be consistent with ServerContextBuilder?

Reply:
Done.